source/OutfitInfoDisplay.cpp

```diff
@@ -635,19 +635,23 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 		attributeValues.emplace_back(Format::Number(arc));
 		attributesHeight += 20;
 	}
-	int homing = outfit.Homing();
-	if(homing)
+	if(outfit.Homing())
 	{
-		static const string skill[] = {
-			"none",
-			"poor",
-			"fair",
-			"good",
-			"excellent"
-		};
-		attributeLabels.emplace_back("homing:");
-		attributeValues.push_back(skill[max(0, min(4, homing))]);
+		attributeLabels.emplace_back("homing type:");
+		attributeValues.push_back(outfit.Leading() ? "leading" : "direct");
```

Comment:
You no longer pass an std::string
```suggestion
		attributeValues.emplace_back(outfit.Leading() ? "leading" : "direct");
```